wiki/Beatmap_Editor/Beat_Snap_Divisor/en.md

```diff
@@ -1,123 +1,113 @@
-# Beat snap divisor
+# Beat Snap Divisor
 
-The Beat Snap Divisor is used to limit where notes can be placed so that they always fall on the beat. Moving the slider to the right increases the granularity of the notes, that is, more notes can be placed in the same measure.
+![Beat Snap Divisor in the editor](img/beat_snap_divisor.png)
 
-Each of the settings put different colours of ticks on the [Timelines](/wiki/Beatmap_Editor/Timelines). The ticks are as follows (for [Standard](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), and [osu!catch](/wiki/Game_mode/osu!catch)):
+The **Beat Snap Divisor** is a built-in editor functionality that ensures every [hit objects](/wiki/Hit_Object) are placed proportionally to the song's beats. It can be found in the upper-right portion of the editor screen, where it corresponds to the granularity of the objects on the [editor timeline](/wiki/Beatmap_Editor/Timelines); that is, a denser Beat Snap Divisor setting allows for more notes can be placed in the same measure and vice versa. 
 
-## osu!, osu!taiko and osu!catch
+The osu! editor supports eleven different Beat Snap Divisor settings as follows:
 
-### Divisor 1/1
+## 1/1 snap divisor
 
-![Divisor 1/1](/wiki/shared/BSD_1_1b.jpg "Divisor 1/1")
+![1/1 snap divisor in the osu!/osu!taiko/osu!catch editor](/wiki/shared/BSD_1_1b.jpg "1/1 snap divisor in the osu!/osu!taiko/osu!catch editor")
 
-- Full Beats
-- White ticks only
-- Best used for Easy difficulty
+![1/1 snap divisor in the osu!mania editor](/wiki/shared/1_1_m.jpg "1/1 snap divisor in the osu!mania editor")
 
-### Divisor 1/2
+The 1/1 snap divisor allows for hit objects to be placed on every *full beats*. It is being represented by white-coloured ticks on the editor timeline.
```

Comment:
```suggestion
The 1/1 snap divisor allows for hit objects to be placed on every *full beat*. It is represented by white-coloured ticks on the editor timeline.
```
usage of "it" like this is not really encouraged. either way, sentence should be written in present and not present progressive (this goes for all other lines like this)